src/lib.rs

```diff
@@ -68,15 +68,20 @@ pub enum DisplayState {
 ///[`set_config`]: struct.HexView.html#method.set_config
 #[derive(Debug, Clone, Copy)]
 pub struct HexViewConfig {
-    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the bytes_per group value. Default is 16
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the `bytes_per_group` value.
+    /// Default is 16.
     pub bytes_per_line: usize,
-    /// Controls the number of bytes per group. It needs to be greater than 0 equal or lower than bytes_per_line. Default is 1.
+    /// Controls the number of bytes per group. It needs to be greater than 0 equal or lower than `bytes_per_line`.
+    /// Default is 1.
```

Comment:
Same here. Please make a newline after the first dot as well as an **empty** line (like above)